src/htmlparser.js

```diff
@@ -52,7 +52,7 @@
   });
 
   // Empty Elements - HTML 4.01
```

Comment:
Hm, we should probably update comments about "HTML 4.01" since we longer constrain ourselves to it.